src/mcl_3dl.cpp

```diff
@@ -614,10 +612,10 @@ class MCL3dlNode
     map_pos.x_ = f_pos_[0]->in(map_pos.x_);
     map_pos.y_ = f_pos_[1]->in(map_pos.y_);
     map_pos.z_ = f_pos_[2]->in(map_pos.z_);
-    trans.setOrigin(tf::Vector3(map_pos.x_, map_pos.y_, map_pos.z_));
-    trans.setRotation(tf::Quaternion(map_rot.x_, map_rot.y_, map_rot.z_, map_rot.w_));
+    tf::vector3TFToMsg(tf::Vector3(map_pos.x_, map_pos.y_, map_pos.z_), trans.transform.translation);
```

Comment:
would be better not to use `tf::` functions.
`#include <tf/transform_datatypes.h>` seems be removable if they are replaced.